tools/distpartitioning/convert_partition.py

```diff
@@ -0,0 +1,448 @@
+import os
```

Comment:
there are two convert_partition.py?

Reply:
convert_partition.py exists in tools/distpartitioning directory. It is removed from tools/ directory.